<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWidget</class>
 <widget class="QWidget" name="MWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/img/046.png</normaloff>:/img/046.png</iconset>
  </property>
  <widget class="QSlider" name="hSlider_music">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>101</y>
     <width>441</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>music:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditMusicName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>36</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>35</y>
     <width>46</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>34</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>step:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MyButton" name="btnPlay">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/img/play.png</normaloff>:/img/play.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyButton" name="btnNext">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/img/next.png</normaloff>:/img/next.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyButton" name="btnSpeedDown">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/img/speedDown.png</normaloff>:/img/speedDown.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyButton" name="btnSpeedUp">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/img/speedUp.png</normaloff>:/img/speedUp.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>140</y>
     <width>29</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="MyButton" name="btnSound">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>90</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/Graphite_Orange_Speaker.ico</normaloff>:/img/Graphite_Orange_Speaker.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="MyButton" name="btnDeal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>51</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/img/055.png</normaloff>:/img/055.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0:0:0/0:0:0</string>
   </property>
  </widget>
  <widget class="Mrepeat" name="hScrollBarStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>513</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Mrepeat" name="hScrollBarStop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>514</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>270</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>start:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>310</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>stop:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckBoxRepeat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>247</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>repeat</string>
   </property>
  </widget>
  <widget class="QLabel" name="labStart">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0:0:0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labStop">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0:0:0</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header>mybutton.h</header>
  </customwidget>
  <customwidget>
   <class>Mrepeat</class>
   <extends>QScrollBar</extends>
   <header location="global">mrepeat.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
